db/tests/types/operations/test_cast.py

```diff
@@ -36,6 +37,9 @@
 NUMERIC = PostgresType.NUMERIC.value.upper()
 REAL = PostgresType.REAL.value.upper()
 SMALLINT = PostgresType.SMALLINT.value.upper()
+DATE = PostgresType.DATE.value.upper()
```

Comment:
`Date` constant has already be initialized at [line 31](https://github.com/centerofci/mathesar/blob/7a5a8da293eed084ffd4bb1bc563a6a6aeb159a0/db/tests/types/operations/test_cast.py#L31)